src/main/python/api.py

```diff
@@ -57,4 +57,4 @@ def get_answers(question, num_hits, k):
     print("Port: {}".format(app.config['port']))
     print("Debug info: {}".format(args.debug))
 
-    app.run(debug=args.debug, host=app.config['host'], port=app.config['port'])
+    app.run(debug=args.debug, host=app.config['host'], port=int(app.config['port']))
```

Comment:
Interesting, I was aware of this, didn't do this conversion, and the server started fine.